install/mysql/glpi-empty.sql

```diff
@@ -6171,6 +6172,7 @@ CREATE TABLE `glpi_queuednotifications` (
   `documents` text,
   `mode` varchar(20) NOT NULL COMMENT 'See Notification_NotificationTemplate::MODE_* constants',
   `event` varchar(255) DEFAULT NULL,
+  `documents_data` varchar(255) DEFAULT NULL,
```

Comment:
255 may be a bit short for namespaced itemtypes.
Also, why not using two distinct fields?
```suggestion
  `itemtype_of_documents` varchar(255) DEFAULT NULL,
  `items_id_of_documents` int unsigned NOT NULL DEFAULT '0',
```